manifests/gcp_marketplace/chart/kubeflow-pipelines/templates/metadata.yaml

```diff
@@ -39,7 +39,7 @@ spec:
         imagePullPolicy: 'Always'
         env:
         # TODO: merge all into mysql-credential
-        {{ if .Values.managedstorage.enabled }}
+        {{ if .Values.managedstorage.cloudsqlInstanceConnectionName }}
         - name: DBCONFIG_USER
```

Comment:
e.x. the case user set the field but didn't check the CheckBox (the UI give user expecation that it won't use CloudSQL).

Let's quickly review tomorrow. (I didn't aware you are still online, thanks)